mapswipe_workers/mapswipe_workers/project_types/base/project.py

```diff
@@ -214,10 +214,13 @@ def save_to_firebase(self, project, groups, groupsOfTasks):
 
         logger.info(f"there are {len(groupsOfTasks)} groups for this project")
         c = 0
-        for group_id in groupsOfTasks.keys():
-            tasks_list = groupsOfTasks[group_id]
-            c += 1
-            if self.projectType in [ProjectType.FOOTPRINT.value]:
+
+        if self.projectType in [ProjectType.FOOTPRINT.value]:
+            # The building footprint project type is the only one
+            # that uses tasks in Firebase.
+            for group_id in groupsOfTasks.keys():
+                tasks_list = groupsOfTasks[group_id]
+                c += 1
```

Comment:
Could this be given a clearer name?

Reply:
I changed it to `group_counter`. We basically just count the number of groups with the variable.